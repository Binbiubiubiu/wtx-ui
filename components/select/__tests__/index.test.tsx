import React from 'react';
import { render } from 'enzyme';
import Select, { Option } from '..';
import toJson from 'enzyme-to-json';

describe('Select', () => {
  it('should render Select components', () => {
    const wrapper = render(
      <>
        <Select>
          <Option value="jack">jack</Option>
          <Option value="lucy">lucy</Option>
          <Option value="yiminghe">yiminghe</Option>
        </Select>
      </>,
    );
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it('should style output is not null', () => {
    expect(require('../style')).not.toBeNull();
  });
});
