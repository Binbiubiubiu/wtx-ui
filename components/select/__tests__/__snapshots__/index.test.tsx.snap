// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Select should render Select components 1`] = `
<div
  class="rc-select rc-select-single rc-select-show-arrow"
>
  <div
    class="rc-select-selector"
  >
    <span
      class="rc-select-selection-search"
    >
      <input
        aria-activedescendant="undefined_list_0"
        aria-autocomplete="list"
        aria-controls="undefined_list"
        aria-haspopup="listbox"
        aria-owns="undefined_list"
        autocomplete="off"
        class="rc-select-selection-search-input"
        role="combobox"
        style="opacity:0"
        value=""
      />
    </span>
    <span
      class="rc-select-selection-placeholder"
    />
  </div>
  <span
    aria-hidden="true"
    class="rc-select-arrow"
    style="user-select:none;-webkit-user-select:none"
    unselectable="on"
  >
    <span
      class="rc-select-arrow-icon"
    />
  </span>
</div>
`;
